src/commcare_cloud/ansible/roles/pgbouncer/defaults/main.yml

```diff
@@ -1,8 +1,10 @@
 pgbouncer_port: 6432
-pgbouncer_pid_file: "/var/run/postgresql/pgbouncer.pid"
+pgbouncer_pid_file: "/var/run/postgresql/pgbouncer-{{ item }}.pid"
```

Comment:
Is `{{ item }}` defined when this file is evaluated? I thought that was local to the playbook being executed, and these are constant default values. I did not realize you could use these as templates to be passed in.

Reply:
Yes, it's gonna be defined when used by the task.

Reply:
@calellowitz ansible does lazy evaluation of template strings so as long as the var is defined when it's needed it should work